cw-orch-daemon/src/keys/private.rs

```diff
@@ -39,19 +39,22 @@ impl PrivateKey {
         secp: &Secp256k1<C>,
         coin_type: u32,
     ) -> Result<PrivateKey, DaemonError> {
-        let phrase = hkd32::mnemonic::Phrase::random(OsRng, hkd32::mnemonic::Language::English);
-
-        PrivateKey::gen_private_key_phrase(secp, phrase, 0, 0, coin_type, "")
+        Self::new_seed(secp, "", coin_type)
     }
     /// generate a new private key with a seed phrase
     pub fn new_seed<C: secp256k1::Signing + secp256k1::Context>(
         secp: &Secp256k1<C>,
         seed_phrase: &str,
         coin_type: u32,
     ) -> Result<PrivateKey, DaemonError> {
-        let phrase = hkd32::mnemonic::Phrase::random(OsRng, hkd32::mnemonic::Language::English);
-
-        PrivateKey::gen_private_key_phrase(secp, phrase, 0, 0, coin_type, seed_phrase)
+        let mut key = [0u8; 16];
+        OsRng.fill_bytes(&mut key);
```

Comment:
Why not reuse built-in rng in bip39? https://docs.rs/bip39/2.0.0/src/bip39/lib.rs.html#290-293

Reply:
good catch, didn't find it